File: 프로그래머스/연습문제/12918_문자열_다루는_기본.cpp
//https://school.programmers.co.kr/learn/courses/30/lessons/12918

// Todo: all_of, isdigit()

#include <string>
#include <algorithm>
#include <cctype>

using namespace std;

bool solution(string s) {
    return (s.size() == 4 || s.size() == 6) && all_of(s.begin(), s.end(), ::isdigit);
}

File: 프로그래머스/연습문제/12918_문자열_다루는_기본.java
//https://school.programmers.co.kr/learn/courses/30/lessons/12918

// Todo: s.chars(), char().allMach(), Charater::isDigit

class Solution {
    public boolean solution(String s) {
        return (s.length() == 4 || s.length() == 6) && s.chars().allMatch(Character::isDigit);
    }
}